lnwallet/btcwallet/psbt.go

```diff
@@ -521,3 +521,39 @@ func (b *BtcWallet) FinalizePsbt(packet *psbt.Packet, accountName string) error
 
 	return b.wallet.FinalizePsbt(keyScope, accountNum, packet)
 }
+
+// lookupFirstCustomAccount returns the first custom account found.
```

Comment:
nit: can use the full 80 character width for comments.

Reply:
strange indeed!